src/view/App.tsx

```diff
@@ -1,21 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-"use strict";
+import { PivotItem } from "office-ui-fabric-react";
 import * as React from "react";
 import "./App.css";
-import Device from "./container/device/Device";
+import { Tab } from "./components/tab/Tab";
+import { DEVICE_LIST_KEY } from "./constants";
+import { Device } from "./container/device/Device";
+
+interface IState {
+    currentDevice?: string;
+}
+
+class App extends React.Component<{}, IState> {
+    state = {
```

Comment:
I'd put this inside the constructor

this.state = initialState

Reply:
Ok, declaring initial state makes it more readable